pyenzyme/tools.py

```diff
@@ -65,7 +67,7 @@ def get_all_parameters(enzmldoc):
     Returns:
         list[Parameter]: A list of all parameters in the EnzymeMLDocument.
     """
-    return find_unique(enzmldoc, target=pe.Parameter)
+    return find_unique(enzmldoc, target=Parameter)
```

Comment:
You can use the parameter array directly. Since the last version, we’ve consolidated all parameters into `enzmldoc.parameters`.